### DISCUSSION TITLE
About plotting mwcs and dtt in frequencies over 1Hz

### DISCUSSION THREAD

**Discussion by sujan1746** [ROBelgium/MSNoise#268] [Q&A] [answered]
Hi,

I computed dv/v in different frequencies such as 0.01-0.05, 0.05-0.1, 0.1-1, 1-5, 5-10, 10-15Hz and used windsorizing 3, 15s and 1s in mwcs_wlen and mwcs_step. (I computed CC in 0.01-15Hz and divided in several frequencies after STACK)

Before using dv/v results, I checked dt/t plots to know whether the blue part is determined well.

I found that under 1Hz, the points in the dt/t plot are aligned so well as you can see below.

This is  the dt/t plot of 0.1-1Hz.
![image](https://user-images.githubusercontent.com/89066297/154389560-c10e9add-ae89-42bb-b831-9931fb87ee86.png)

However, over 1Hz, the points became scattered and it is hard to say they are aligned.

This is the dt/t plot of 1-5Hz.

![image](https://user-images.githubusercontent.com/89066297/154389705-90c14a08-3b0a-48a2-a093-c35f6dd25679.png)

I am not sure whether I can use the dv/v results over 1Hz...

I computed again with using 10s and 0.5s in mwcs_wlen and mwcs_step but still same results.

I would find more the way to figure out and compute again using 20s and 4s in mwcs_wlen and mwcs_step but I am not sure it would work..

**Comment by asyates** [comment, marked as answer]
Hi,

In my opinion, it is probably correct to say that the 1-5 Hz results are not reliable if the dt/t plots generally look like the one you have shown. You could probably try smaller moving windows (mwcs_wlen) for higher frequencies... but i would be surprised if this makes a significant difference. 

I can see that this is a correlation between pairs of stations i.e. TP01 and TP11. What kind of interstation distances are you working with? The distance between stations has a large influence on the frequencies you can expect to propagate coherently between them (i.e. higher frequencies attenuate more).

If you suspect it is an issue of stations being too far apart, and you want to study higher frequencies still, you could try processing single-station dv/v (if you have three component data). You can then, in theory, process higher frequencies at earlier lag times.

**Reply by John8974** [reply]
Thank you for your comment! The interstation distance with station TP01 and TP11 is about 28km. The interstation distances what I have is about 7.3km in minimum, 65km in maximum. The way to process single-station is good. If  I need to use higher frequency, I would use the way.

**Reply by asyates** [reply]
Ah yes, I suspect then that the interstation distance is the primary cause of poor results above 1 Hz (28 km is quite a large distance). You could check how the results look for smaller values of interstation distance (e.g. closer to the minimum)... but yes, I would say your best bet will probably be to use single-station if targeting higher frequencies here.

**Reply by John8974** [reply]
I see. I worried about the way to research about higher frequencies in the future but find the solution. One more curious thing is whether it is possible to process single-station dv/v with only vertical component or need all three component data.

**Reply by asyates** [reply]
Hi, yes it is possible. You can compute the auto-correlation ZZ. Note though that we can't whiten our data for auto-correlations (at least, not brutal whitening where the amplitude is set to 1.0) as the only information is in the amplitude spectrum... so working with the cross-components can give cleaner results. That said, less aggressive whitening options should be okay. Personally haven't tested them, but you can check out some recent publications using auto-correlations (i think a smoothed whitening is applied instead).

**Reply by John8974** [reply]
I got it. Thank you for letting me know :)
